ph5/utilities/noven.py

```diff
@@ -13,18 +13,15 @@
 from ph5.utilities import novenqc, novenkef
 
 
-PROG_VERSION = '2018.268'
+PROG_VERSION = '2021.84'
 LOGGER = logging.getLogger(__name__)
 
 try:
-    from PyQt4 import QtGui, QtCore
+    from PySide2 import QtWidgets, QtCore
 except Exception:
-    msg = ("\n\nNo module named PyQt4. "
-           "Please install PyQt4 first, it is needed for noven. "
-           "\n\n"
-           "If using Anaconda run 'conda install pyqt=4'"
-           "For pip users, PyQt4 installation instructions are available at "
-           "http://pyqt.sourceforge.net/Docs/PyQt4/installation.html.")
+    msg = ("\n\nNo module named PySide2. "
+           "Please environment_gui.yml to install conda environment"
```

Comment:
"Please use environment_gui.yml to install all required dependencies..." or something similar to that.